pkg/orchestrator/config/config_test.go

```diff
@@ -118,6 +120,47 @@ func (suite *YamlConfigTestSuite) TestExtractOrchestratorEndpointsPrecedence() {
 	}
 }
 
+func (suite *YamlConfigTestSuite) TestEnvConfigDDURL() {
+	ddOrchestratorURL := "DD_ORCHESTRATOR_EXPLORER_ORCHESTRATOR_DD_URL"
+	expectedValue := "123.datadoghq.com"
+	os.Setenv(ddOrchestratorURL, expectedValue)
+	defer os.Unsetenv(ddOrchestratorURL)
+
+	orchestratorCfg := NewDefaultOrchestratorConfig()
+	err := orchestratorCfg.Load()
+	suite.NoError(err)
+
+	suite.Equal(expectedValue, orchestratorCfg.OrchestratorEndpoints[0].Endpoint.Path)
+}
+
+func (suite *YamlConfigTestSuite) TestEnvConfigMessageSize() {
+	ddMaxMessage := "DD_ORCHESTRATOR_EXPLORER_MAX_PER_MESSAGE"
+	expectedValue := "50"
+	os.Setenv(ddMaxMessage, expectedValue)
+	defer os.Unsetenv(ddMaxMessage)
+
+	orchestratorCfg := NewDefaultOrchestratorConfig()
+	err := orchestratorCfg.Load()
+	suite.NoError(err)
+
+	i, err := strconv.Atoi(expectedValue)
+	suite.NoError(err)
+	suite.Equal(i, orchestratorCfg.MaxPerMessage)
+}
+
+func (suite *YamlConfigTestSuite) TestEnvConfigSensitiveWords() {
+	ddSensitiveWords := "DD_ORCHESTRATOR_EXPLORER_CUSTOM_SENSITIVE_WORDS"
+	expectedValue := "123.datadoghq.com"
```

Comment:
nit: could we use a more realistic value for this test? like "foo bar" as it's a list of word space separated?

Reply:
Makes sense. Updated the test. Additionally I added one edge case for the message if the int is too high, so we default to the default.